app/components/code-sample/bootstrap.yml

```diff
@@ -0,0 +1,5 @@
+_components:
+  code-sample:
+    code: ''
+    language: 'javascript'
+    html: ''
```

Comment:
Enable `Final Newline` option in your editor so it always adds a new line after saving the file